.licenserc.yaml

```diff
@@ -59,6 +59,7 @@ header:
     - '**/*.pem'
     - 'ssh_config'
     - 'workers'
+    - 'streampark-console/streampark-console-webapp'
```

Comment:
It seems that we skip the license checks for all the `streampark-console-webapp` folder, why?

Reply:
> It seems that we skip the license checks for all the `streampark-console-webapp` folder, why?

some files copy from other projects (MIT license), the apache license header cannot be added for these files.  without apache license header will cause the license check fail, so, we skip the License check first.

Reply:
I know this background, but it should be a fine-grain exclusion instead of such a wildcard, as long as we do write our own code under this folder.

Reply:
So we need to list every file？